### PR TITLE
Update to Kotlin 1.0.0. + Json Remote Command

### PR DESCRIPTION
**Pull request by tamayok**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -16,7 +16,4 @@ org.gradle.jvmargs=-Xmx1536m
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
-android.enableJetifier=true
-
-AWS_ACCESS_KEY="DUMMY-CREDENTIALS"
-AWS_SECRET_KEY="DUMMY-CREDENTIALS"
\ No newline at end of file
+android.enableJetifier=true
```
Comment (jameskeith) [inline gradle.properties:19]: removing the dummy credentials will cause build issues for anyone who does not have the AWS_ACCESS_KEY/AWS_SECRET_KEY set in their global `~/.gradle/gradle.properties` file
If we're removing these, can we comment out the `repositories { }` block in `firebase/build.gradle` too?

**Review by jameskeith — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (tamayok) [reply gradle.properties:19]: Good call. I'll comment out that block in the gradle file. I believe we do this in main Kotlin lib as well.

File: firebase/src/main/java/com/tealium/remotecommands/firebase/FirebaseConstants.java [hunk 2]
```diff
@@ -28,7 +28,8 @@ private Keys() {
         public static final String ANALYTICS_ENABLED = "firebase_analytics_enabled";
         public static final String LOG_LEVEL = "firebase_log_level";
         public static final String EVENT_NAME = "firebase_event_name";
-        public static final String EVENT_PARAMS = "firebase_event_params";
+        public static final String EVENT_PARAMS = "event";
+        //        public static final String EVENT_PARAMS = "firebase_event_params";
```
[the remark this answers is not in the document]
Comment (tamayok) [reply firebase/src/main/java/com/tealium/remotecommands/firebase/FirebaseConstants.java:32]: Addressed.

**Review by jameskeith — APPROVED** [review]
lgtm

**Pull request merged**